packages/react/src/components/Header/HeaderActionGroup.jsx

```diff
@@ -205,7 +209,7 @@ const HeaderActionGroup = ({ actionItems, i18n }) => {
                 item={item}
                 index={i}
                 key={`header-action-item-${item.label}-${i}`}
-                testID={`header-action-item-${item.label}`}
+                testId={`header-action-item-${item.label}`}
```

Comment:
Do we have test for the `Header` components?

Reply:
we do now!